prometheus_hardware_exporter/collectors/redfish.py

```diff
@@ -38,28 +48,55 @@ def get_sensor_data(self, host: str, username: str, password: str) -> Dict[str,
             output[name] = sensors
         return output
 
-    def _get_sensor_data(self, host: str, username: str, password: str) -> Optional[List[Any]]:
+    def _get_sensor_data(
+        self, login_info: dict, timeout: int, max_retry: int
+    ) -> Optional[List[Any]]:
         """Return sensor if sensor exists else None."""
```

Comment:
Can you also describe the parameters(especially login_info) in the docstring? Since it's now a dict, it's not very intuitive.

Reply:
Good suggestion, adding for both the sensor methods.